modules/logwriter/variables.tf

```diff
@@ -17,13 +17,18 @@ variable "bucket_arn" {
   EOF
   type        = string
   nullable    = false
+
+  validation {
+    condition     = can(regex("^arn:.*", var.bucket_arn))
+    error_message = "ARN is invalid."
+  }
 }
 
 variable "prefix" {
   description = "Optional prefix to write log records to."
   type        = string
   nullable    = false
-  default     = "observe"
```

Comment:
Curious why we switched from `observe` here. Wouldn't that be easier to identify and troubleshoot within customer environments?

Reply:
No, we don't actually use `observe` in practice, since the `stack` module always sets this to `cloudwatchlogs/`. We are reliant on that path in order to infer the content type: https://github.com/observeinc/aws-sam-apps/blob/main/handler/forwarder/override/presets/aws/v1.yaml#L1-L6